lib/datasets/kuzushiji-mnist.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'mnist'
+
+module Datasets
+  class KuzushijiMNIST < MNIST
+    BASE_URL = "http://codh.rois.ac.jp/kmnist/dataset/kmnist/"
+
+    private
+    def dataset_name
+      "Kuzushiji-MNIST"
+    end
+
+    def licenses
+      ["CC BY-SA 4.0"]
```

Comment:
Could you use SPDX license ID https://spdx.org/licenses/ ?

```suggestion
      ["CC-BY-SA-4.0"]
```